rules-java/src/main/resources/reports/templates/java_application.ftl

```diff
@@ -66,17 +66,45 @@
         <div class="panel-heading">
             <h3 class="panel-title">${projectModel.rootFileModel.prettyPath}</h3>
         </div>
-        <div class="container-fluid summaryMargin">        	
+        <div class="container-fluid summaryMargin">
             <div class='col-md-3 text-right totalSummary'>
                 <div class='totalLoe'>
                     ${getMigrationEffortPoints(projectModel, false)}
                 </div>
                 <div class='totalDesc'>Story Points</div>
             </div>
-            
+
             <div class='col-md-6 pull-right windupPieGraph archiveGraphContainer'>
                 <div id="project_${projectModel.asVertex().getId()?string("0")}_pie" class='windupPieGraph'></div>
             </div>
+
+			<div class="col-md-6 pull-right">
+				<table class="table">
+					<tr>
+						<th>Vendor</th>
```

Comment:
Label instead of Vendor? I guess Vendor is probably OK on this part of the report.